eng/Versions.props

```diff
@@ -305,6 +305,12 @@
     <UsingToolMicrosoftNetCompilers Condition="'$(BootstrapBuildPath)' == ''">true</UsingToolMicrosoftNetCompilers>
     <UseVSTestRunner>true</UseVSTestRunner>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(NoPdbConversion)' == 'true'">
+    <UsingToolPdbConverter>false</UsingToolPdbConverter>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(NoVsixGeneration)' == 'true'">
```

Comment:
>NoVsixGeneration [](http://example.com/codeflow?start=31&length=16)

The extra property is unnecessary. Setting /p:UsingToolVSSDK=false on msbuild command line overrides all values set in targets files.